gslib/commands/test.py

```diff
@@ -554,6 +554,19 @@ def RunCommand(self):
         elif o == '-u':
           tests.util.RUN_INTEGRATION_TESTS = False
 
+    if tests.util.RUN_INTEGRATION_TESTS:
+      try:
+        tests.util.AuthorizeProjectToUseTestingKmsKeys()
+        service_agent_authorized_successfully = True
+      except Exception as error:
+        # Authorized keys are static resources that typically already have
+        # appropriate permissions. Demoting errors to warnings means transient
+        # errors will not block running the whole suite of tests.
+        logging.warning(
+            'Failed to authorize the service agent to use KMS keys.')
```

Comment:
This warning still sounds like a non-transient error. Do we even need it with the new logging below?

Reply:
Removed the warning as part of a change in approach.